spotseeker_server/test/images/oauth_spot_info.py

```diff
@@ -1,13 +1,14 @@
-# Copyright 2022 UW-IT, University of Washington
+# Copyright 2023 UW-IT, University of Washington
 # SPDX-License-Identifier: Apache-2.0
 
+import secrets
```

Comment:
Looks like an unused secret?

Reply:
entire file is removed in task/oauth-tests